query/src/main/java/org/infinispan/query/Search.java

```diff
@@ -106,10 +106,16 @@ private static void checkBulkReadPermission(AdvancedCache<?, ?> cache) {
       }
    }
 
+   /**
+    * Returns query statistics for the cache.
+    */
    public static <K, V> SearchStatistics getSearchStatistics(Cache<K, V> cache) {
       return ComponentRegistryUtils.getSearchStatsRetriever(cache).getSearchStatistics();
    }
 
+   /**
+    * Returns query statistics for the cluster.
+    */
```

Comment:
Please change to "Returns aggregated search statistics for all nodes of the cluster"